AABB_tree/demo/AABB_tree/AABB_demo.cpp

```diff
@@ -23,14 +23,24 @@
 
 int main(int argc, char **argv)
 {
+  QSurfaceFormat fmt;
```

Comment:
Why here you don't use code_to_call_before_creation_of_QCoreApplication ?

Reply:
At some point I realized it was the same. The point is to set the default before the context is created, but not especially after the application is launched.